com.github.pyfa_org.Pyfa.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.pyfa_org.Pyfa</id>
+  <launchable type="desktop-id">com.github.pyfa_org.Pyfa.desktop</launchable>
+  <name>Pyfa</name>
+  <developer_name>pyfa-org</developer_name>
+  <summary>Python fitting assistant for EVE: Online</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-3</project_license>
```

Comment:
this is an invalid SPDX license identifier.

Reply:
Fixed. Also changed metadata license to CC0-1.0